Add `supports_pdf_input: true` for specific Bedrock Claude models

iwamot opened the pull request:
Added `"supports_pdf_input": true` for the following Bedrock models in `model_prices_and_context_window.json`.

- `bedrock/anthropic.claude-3-haiku-20240307-v1:0`
- `bedrock/anthropic.claude-3-sonnet-20240229-v1:0`
- `bedrock/anthropic.claude-3-5-haiku-20241022-v1:0`
- `bedrock/anthropic.claude-3-5-sonnet-20241022-v2:0`
- `bedrock/anthropic.claude-3-5-sonnet-20240620-v1:0`

PDF input has been tested locally and confirmed to work.

iwamot committed:
Add `supports_pdf_input: true` for specific Bedrock Claude models

krrishdholakia commented:
Hey @iwamot can you add this for the cross region endpoints as well? 

i believe we have a unit test that'll fail due to inconsistencies between base model and cri model

iwamot commented:
@krrishdholakia Thanks for the review! I'll add it for the cross region endpoints and check if the tests pass.

iwamot committed:
Add `supports_pdf_input: true` for specific Bedrock Claude cross-region endpoints

iwamot commented:
@krrishdholakia I've added it for the cross region endpoints and also fixed indentation. Sorry, I couldn't run the tests since I haven't set up the environment properly. Let me know if any changes are needed!

ChrisjanWust commented:
Was about to write a PR myself, but I see it's already here. This looks good - can we merge?
